docs/00_Prepare_everything.md

```diff
@@ -1,86 +1,82 @@
+5. In the next window, select "__Use gradle 'wrapper' task configuration__".
 ![use gradle 'wrapper' task configuration checkbox](img/select_gradle_wrapper.png)
 
-### Running test in Android Studio/IntelliJ IDEA
+### Running "test" in Android Studio/IntelliJ IDEA
 
-Go to _run_ > _edit configurations_ > _add new configuration_ and select "Gradle".
-As Gradle Project, select NewPipeExtractor. As task add "test". Now save and you should be able to run.
+Go to _Run_ > _Edit Configurations_ > _Add New Configuration_ and select "Gradle".
+As Gradle Project, select NewPipeExtractor. As a task, add "test". Now save and you should be able to run.
 
 ![tests passed on idea](img/prepare_tests_passed.png)
 
-# Inclusion criteria for services
+# Inclusion Criteria for Services
 
-After creating you own service you will need to submit it to our [NewPipeExtractor](https://github.com/teamnewpipe/newpipeextractor)
- repository. However in order to include your changes you need to follow these rules:
+After creating you own service, you will need to submit it to our [NewPipeExtractor](https://github.com/teamnewpipe/newpipeextractor)
+ repository. However, in order to include your changes, you need to follow these rules:
 
 1. Stick to our [Code contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#code-contribution)
-2. Do not send services that present content we [don't allow](#not-allowed-content) on NewPipe.
-3. You need to be willing to keep on maintaining your service after submission.
-4. Be patient and do the requested changes when one of our maintainers rejects your code.
+2. Do not send services that present content we [don't allow](#content-that-is-not-permitted) on NewPipe.
+3. You must be willing to maintain your service after submission.
+4. Be patient and make the requested changes when one of our maintainers rejects your code.
 
-## Allowed Content
+## Content That is Permitted:
 
-- Basically anything except [NOT allowed content](#not-allowed-content).
-- Any kind of porn/NSFW that is allowed according to the [US Porn act](https://www.justice.gov/archive/opa/pr/2003/April/03_ag_266.htm).
-- Advertisement (may be handled specially though)
+- Any content that is not in the [list of prohibited content](#content-that-is-not-permitted).
+- Any kind of pornography or NSFW content that does not violate US law.
+- Advertising, which may need to be approved beforehand.
 
-## NOT allowed Content
+## Content That is NOT Permitted:
 
-- NSFL (Not Safe For Life)
-- Porn that is not allowed according to [US Porn act](https://www.justice.gov/archive/opa/pr/2003/April/03_ag_266.htm).
-- Any form of violence
-- Child pornography
-- Media that harms others
-- Media that violates human rights
-- Copyright infringement/pirated media
+- Content that is considered NSFL (Not Safe For Life)
+- Content that is prohibited by US federal law (Sexualization of minors, etc).
```

Comment:
@TheAssassin or @theScrabi can you please take a look at this. I'd suggest to keep the content of the three bullet points.
```suggestion
- Content that is prohibited by US federal law (Sexualization of minors, any form of violence, violations of human rights, etc).
```